network/p2p/ATTESTER.md

```diff
@@ -0,0 +1,74 @@
+# Attester message validation
+
+## Message reactions
+
+Node can react to a message in three ways:
+- `Accept` - message is valid, accepted and processed further.
+- `Ignore` - message is invalid, ignored and not processed further, violation is noted.
+- `Reject` - message is rejected, ignored and not processed further, peer is banned.
+
+## Attester assertions
+
+- Message validator maintains a state for each validator.
+- Validator attests only once per epoch.
```

Comment:
lets add a comment need to check if upon reorg can be more than one duty per epoch

Reply:
added

Reply:
What you mean as the necessity for a second attestation is like the following scenario?
- validator V attests for block B at slot S
- a fork happens in slot S-1 and B is not in the canonical chain
- V should make another attestation for the canonical block of slot S?

But wouldn't this have the risk of being considered a slashable attestation?
- (Slashable Offences section) https://ethos.dev/beacon-chain#:~:text=An%20attestation%20contains%20both%20an,chain%20in%20a%20single%20epoch
- https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#is_slashable_attestation_data